---
title: "&#39; 마지막으로 &#39; 외부에 나타날 수 없습니다는 &#39; Try &#39; 문"
ms.date: 07/20/2015
ms.prod: .net
ms.technology: devlang-visual-basic
ms.topic: article
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords: BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
caps.latest.revision: "7"
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: 259df1345b443d5023ebecd457c81167d0799fe0
ms.sourcegitcommit: 34ec7753acf76f90a0fa845235ef06663dc9e36e
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 12/21/2017
---
# <a name="39finally39-cannot-appear-outside-a-39try39-statement"></a>&#39; 마지막으로 &#39; 외부에 나타날 수 없습니다는 &#39; Try &#39; 문
`Finally` 는 `Try...Catch...Finally` 블록을 완료하는 데 사용되므로 블록의 끝에 한 번만 나타날 수 있습니다. 불필요한 `Finally`가 있거나, `Finally` 문이 해당 `Try` 블록의 경계 외부에 나타납니다.  
  
 **오류 ID:** BC30382  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
1.  제거할 때 불필요 한 `Finally` 문.  
  
2.  `Finally` 문을 코드의 적절한 위치로 이동합니다.  
  
## <a name="see-also"></a>참고 항목  
 [Try...Catch...Finally 문](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
